content/en/containers/kubernetes/appsec.md

```diff
@@ -0,0 +1,360 @@
+---
+title: Kubernetes Application Security - Appsec Injector
```

Comment:
```suggestion
title: Kubernetes Application Security using Appsec Injector
```

Reply:
Appsec is almost always spelled as AppSec.
Is there a reason we are spelling it as Appsec?